### PR TITLE
added more logging

### PR DESCRIPTION
**Pull request by joycechangeorg**
@edgeworthch @mdimas @kamalama 

Adding the summary of the records that caused 429 or 413 error to assist with debugging

### CODE REVIEW

File: lib/fluent/plugin/out_amplitude.rb [hunk 1]
```diff
@@ -189,22 +207,47 @@ def send_to_amplitude(records)
...
-          duration: res.total_time * 1000
+          duration: res.total_time * 1000,
+          user_id_counts: user_id_counts.to_s
         )
         raise AmplitudeError, "Error: #{error_string}"
       end
     end
 
-    def log_error(error)
+    def error_postsend_fail(code, body, records, duration, user_id_counts)
```
Comment (edgeworthch) [inline lib/fluent/plugin/out_amplitude.rb]: I'm not sure this interface matches what you have above—if I'm reading it correctly, a single hash is being passed in, but you're expecting five different arguments here.  Did you test this locally already?

Comment (ysgard) [reply lib/fluent/plugin/out_amplitude.rb]: Has it been changed?  I see five arguments being passed into `error_postsend_fail`

**Review by ysgard — CHANGES_REQUESTED** [review]
I share Edgeworth's concern that we're flooding the metrics with `user_id`, unless `user_id` means something different in this context.

Has this been tested?

**Comment by willbarrett** [comment]
Closing as something we shouldn't do.

**Pull request closed without merging**